sdk/pinocchio/src/entrypoint/mod.rs

```diff
@@ -157,6 +163,13 @@ macro_rules! program_entrypoint {
     };
 }
 
+/// Align a pointer to the BPF alignment of `u128`.
+macro_rules! align_pointer {
+    ($ptr:ident) => {
+        (($ptr as usize + (BPF_ALIGN_OF_U128 - 1)) & !(BPF_ALIGN_OF_U128 - 1)) as *mut u8
+    };
+}
```

Comment:
Why does this align to `u128`? Shouldn't it align to `u64` per the serialization spec?